content/en/docs/advanced/creating_custom_cloud_images.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Creating Custom Cloud Images"
+linkTitle: "Creating Custom Cloud Images"
+weight: 1
+description: A comprehensive guide to creating custom cloud images with Kairos using the latest tools
+---
+
+This guide provides a complete walkthrough for creating custom cloud images with Kairos. It covers the entire process from start to finish, using the latest tools like `kairos-init` and `AuroraBoot`.
+
+## Overview
+
+Kairos provides several tools to create custom cloud images:
+
+1. **kairos-init**: A tool for creating base container images
+2. **AuroraBoot**: A tool for generating bootable images (ISOs, cloud images, etc.)
+3. **Customization tools**: Various methods to customize the images
+
+## Prerequisites
+
+Before starting, ensure you have:
+
+- Docker installed and running
+- A Linux machine with KVM support (for testing)
+- Basic understanding of container images and cloud configurations
+
+## Step 1: Creating a Base Image with kairos-init
+
+The first step is to create a base container image using [`kairos-init`](https://github.com/kairos-io/kairos-init). This tool allows you to create a custom base image from popular distributions.
+
+Here's a basic example of creating a base image:
+
+```bash
+docker build -t my-custom-image - <<EOF
+ARG BASE_IMAGE=ubuntu:24.04
+
+FROM quay.io/kairos/kairos-init:v0.4.3 AS kairos-init
+
+FROM ${BASE_IMAGE} AS base-kairos
+ARG VARIANT=core
+ARG MODEL=generic
+ARG TRUSTED_BOOT=false
+ARG KUBERNETES_DISTRO=k3s
+ARG KUBERNETES_VERSION=latest
+ARG FRAMEWORK_VERSION=v2.20.0
+ARG VERSION
+
+COPY --from=kairos-init /kairos-init /kairos-init
+RUN /kairos-init -f "${FRAMEWORK_VERSION}" -l debug -s install -m "${MODEL}" -v "${VARIANT}" -t "${TRUSTED_BOOT}" -k "${KUBERNETES_DISTRO}" --k8sversion "${KUBERNETES_VERSION}" --version "${VERSION}"
+RUN /kairos-init -f "${FRAMEWORK_VERSION}" -l debug -s init -m "${MODEL}" -v "${VARIANT}" -t "${TRUSTED_BOOT}" -k "${KUBERNETES_DISTRO}" --k8sversion "${KUBERNETES_VERSION}" --version "${VERSION}"
+RUN /kairos-init -f "${FRAMEWORK_VERSION}" -l debug --validate -m "${MODEL}" -v "${VARIANT}" -t "${TRUSTED_BOOT}" -k "${KUBERNETES_DISTRO}" --k8sversion "${KUBERNETES_VERSION}" --version "${VERSION}"
+RUN rm /kairos-init
+EOF
+```
+
+## Step 2: Customizing the Image
+
+Once you have a base image, you can customize it by:
+
+1. Adding packages
+2. Modifying configurations
+3. Adding custom services
+4. Including additional files
+
+See the [Customizing Images]({{< ref "/docs/advanced/customizing" >}}) guide for detailed instructions.
+
+## Step 3: Building Bootable Images with AuroraBoot
+
+After creating and customizing your base image, you can use AuroraBoot to create bootable images:
+
+```bash
+docker run -v "$PWD"/build:/tmp/auroraboot \
+             -v /var/run/docker.sock:/var/run/docker.sock \
+             --rm -ti quay.io/kairos/auroraboot \
```

Comment:
no version will default to latest, do we want that?

Reply:
Switched to use existing variable.